service/mount.go

```diff
@@ -180,7 +180,7 @@ func mkdir(ctx context.Context, path string) (bool, error) {
 	// Fetch log handler
 	st, err := os.Stat(path)
 	if os.IsNotExist(err) {
-		if err := os.Mkdir(path, 0750); err != nil {
+		if err := os.Mkdir(path, 0o750); err != nil {
```

Comment:
When specifying octal permissions in Go, it is advised to use the `0o` prefix to explicitly indicate that the literal is in octal form. This is to make the code more readable and to avoid confusion, especially for developers who might not be familiar with this convention.

In Go, the `0o` prefix was introduced to make it clear that the number following it is intended to be interpreted in octal form. It enhances code readability and reduces the chances of mistakes or misinterpretations.

Reply:
Thanks for the details .